docs/src/using-criticalup/creating-sample-manifest.rst

```diff
@@ -0,0 +1,33 @@
+.. SPDX-FileCopyrightText: The Ferrocene Developers
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+
+.. _creating_sample_manifest:
+
+Creating sample manifest
+========================
+
+This chapter describes a way to quickly create a sample ``criticalup.toml`` for your project.
+
+Creating a sample manifest file
+-------------------------------
+
+Running the ``init`` command will create a file ``criticalup.toml`` in the current working directory.
+
+.. code-block::
+
+   criticalup init --release <RELEASE>
+
+
+The ``--release`` flag's value must be one of the product releases listed on the `release
+channels page <https://releases.ferrocene.dev/ferrocene/index.html>`_.
+
+Running this command again will overwrite the existing ``criticalup.toml`` file in the current working directory.
```

Comment:
Perhaps it should error if one already exists?

Reply:
Added an error. Do you think we can have a flag `--overwrite` when passed won't show error and just overwrite the existing file?

Reply:
Let's no overcomplicate things, `rm criticalup.toml` exists.